admin_manual/configuration/files/encryption_configuration.rst

```diff
@@ -456,45 +370,45 @@ There may be other files that are not encrypted; only files that are exposed to
-You must run *occ* as your HTTP user. See :doc:`../../configuration/server/occ_command` to learn more about *occ*.
+Also, share owners may need to re-share files after encryption is enabled. 
+Users who are trying to access the share will see a message advising them to ask the share owner to re-share the file with them. 
 
-Encryption migration to ownCloud 8.1
-------------------------------------
+For individual shares, un-share and re-share the file. 
+For group shares, share with any individuals who can't access the share. 
+This updates the encryption, and then the share owner can remove the individual shares.
 
-The encryption backend has changed again in ownCloud 8.1, so you must take some additional steps to migrate encryption correctly. 
-If you do not follow these steps you may not be able to access your files.
+.. figure:: images/encryption9.png
 
-Before you start your upgrade, put your ownCloud server into ``maintenance:singleuser`` mode (See :doc:`../../maintenance/enable_maintenance`.) 
-You must do this to prevent users and sync clients from accessing files before you have completed your encryption migration.
+How To Enable Encryption From the Web-UI
+----------------------------------------
 
-After your upgrade is complete, follow the steps in :ref:`enable_encryption_label` to enable the new encryption system. 
-Then click the **Start Migration** button on your Admin page to migrate your encryption keys, or use the ``occ`` command. 
-We strongly recommend using the ``occ`` command; the **Start Migration** button is for admins who do not have access to the console, for example, installations on shared hosting. 
-This example is for Debian/Ubuntu Linux::
+1. First, you must enable the encrypton app, and then select an encryption type. 
+Go to the **Apps** section of your Admin page, click on **Show disabled Apps** and enable **Default encryption module**. 
 
- $ sudo -u www-data php occ encryption:migrate
- 
-This example is for Red Hat/CentOS/Fedora Linux::
+2. After that go to the encryption section of your Admin page, and ckeck the checkbox "Enable server-side encryption".
 
- $ sudo -u apache php occ encryption:migrate
- 
-You must run ``occ`` as your HTTP user; see 
-:doc:`../../configuration/server/occ_command`.
+3. Then select an encryption Type. Masterkey and User-key are your option. Masterkey is recommended. 
 
-When you are finished, take your ownCloud server out of 
-``maintenance:singleuser`` mode.
+4. Now you **must** log out and then log back in to initialize your encryption keys.
 
-.. Links
-   
-.. _file-level or whole-disk encryption: http://blog.vormetric.com/2015/06/23/locking-down-data-full-disk-encryption-vs-file-level-encryption/
+When you log back in, a checkbox for enabling encryption on your home storage, will now be available — checked by default. 
+Uncheck it to avoid encrypting your home storage.
```

Comment:
A bit of explanation here?
Normally users have just asked to encrypt stuff, so they would not be wanting to uncheck the box???

Reply:
Don't know, did not write that, just restructuring.

Reply:
I think this is the option to opt-out. 

It may seem unnecessary for some, but necessary for others.

Also this is merely the explanation of this option.

What would you like for me to write as an explanation there?

Reply:
Good question - the sentence came as a bit of a surprise to me when I read it. Maybe:
```
If you have changed your mind, and do not want to encrypt your home storage, then you  can uncheck the checkbox.
```
And I wonder what happens if you do that? Hopefully the encryption settings that you made are now forgotten.

Reply:
Well if I read the document correctly, 

1. the main purpose of encryption is to keep your files encrypted on remote storages, like google drive.
2. you can separately choose to encrypt remote files or/and your home storage.

those infos are at the top of the document, that huge text that no one reads.

That said, given that the checkbox is checked per default. I am also not sure what happens if you un-check this.

we should ask  @sharidas or @PVince81

Reply:
tested - this is a oc 9 option. in 10.0.8 this option does not exist

Reply:
OK - so you are not "changing your mind", you are choosing to encrypt or not the home storage part of your files, and the other remote storages will be encrypted like you previously asked.
If the option is gone in 10.0.something then the doc needs updating anyway.

Reply:
updated already